### PR TITLE
Modelops61

### PR DESCRIPTION
**Pull request by pabloesc**
ModelOps - Import and Deploy your first BYOM Model
Learn how to import and deploy a BYOM model type in Vantage to accelerate the operationalization and monitor efficiently the deployment.

### CODE REVIEW

File: modules/ROOT/nav.adoc [hunk 1]
```diff
@@ -14,6 +14,7 @@
 ** xref:tools-and-utilities:run-bulkloads-efficiently-with-teradata-parallel-transporter.adoc[Load data with TPT]
 ** xref::perform-time-series-analysis-using-teradata-vantage.adoc[Perform time series analysis]
 ** xref::create-parquet-files-in-object-storage.adoc[]
+** xref::modelops-byom.adoc[]
```
[the remark this answers is not in the document]
Comment (pabloesc) [reply modules/ROOT/nav.adoc:17]: ok good suggestion. I updated the name of the file and the nav.adoc

File: modules/ROOT/pages/modelops-byom.adoc [hunk 2]
```diff
@@ -0,0 +1,251 @@
+= ModelOps - Import and Deploy your first BYOM Model
+:experimental:
+:page-author: Pablo Escobar de la Oliva
+:page-email: pablo.escobardelaoliva@teradata.com
+:page-revdate: October 5th, 2022
+:description: Tutorial for deploying and monitor a PMML model into Vantage using ModelOps
+:keywords: modelops, clearscape analytics, teradata, data warehouses, teradata, vantage, cloud data platform, machine learning, artificial intelligence, business intelligence, enterprise analytics
```
[the remark this answers is not in the document]
Comment (pabloesc) [reply modules/ROOT/pages/modelops-byom.adoc:7]: modelops is the first keword, I will include byom and python as well

File: modules/ROOT/pages/modelops-byom.adoc [hunk 3]
```diff
@@ -0,0 +1,251 @@
...
+
+* Path: https://github.com/Teradata/modelops-demo-models
+
+* Credentials: No Credentials
+
+* Branch: master
+
+Here you can test the git connection, If is green then save and continue. For the Service connection you can skip for now. 
+
+When Creating a new Project, ModelOps will ask you for a new connection.
```
[the remark this answers is not in the document]
Comment (pabloesc) [reply modules/ROOT/pages/modelops-byom.adoc:71]: ok i will have a look

**Comment by pabloesc** [comment]
Fixes done and new quickstart for GIt models included. A new partial doc is been created for the duplicated parts

**Comment by pabloesc** [comment]
ModelOps - Import and Deploy your first GIT Model
Learn how to import and deploy a Python (based in GIT repository) model type in Vantage to accelerate the operationalization and monitor efficiently the deployment.

**Review by adamtwo — APPROVED** [review]

**Pull request merged**